easybuild/easyconfigs/m/MotionCor2/MotionCor2-1.4.4-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,46 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Ake Sandgren
+# HPC2N
+# Umea University
+
+name = 'MotionCor2'
+version = '1.4.4'
+
+homepage = 'https://msg.ucsf.edu/'
+description = """MotionCor2 correct anisotropic image motion at the
+single pixel level across the whole frame, suitable for both single
+particle and tomographic images. Iterative, patch-based motion detection
+is combined with spatial and temporal constraints and dose weighting.
+
+Cite publication: Shawn Q. Zheng, Eugene Palovcak, Jean-Paul Armache,
+Yifan Cheng and David A. Agard (2016) Anisotropic Correction of
+Beam-induced Motion for Improved Single-particle Electron
+Cryo-microscopy, Nature Methods, submitted.
+BioArxiv: https://biorxiv.org/content/early/2016/07/04/061960
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+sources = [{
+    'filename': '%(name)s_%(version)s.zip',
+    'extract_cmd': 'unzip -j %s'
+}]
```

Comment:
Now with the new motioncor easyblock this should be changed back to the simple source specification used in older versions.